src/main/java/pageobject/LoginPage.java

```diff
@@ -0,0 +1,50 @@
+package pageobject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends AbstractPage {
+    private static final String URL = "https://network-international--qa.cs83.my.salesforce.com/";
+    private static final By USER_NAME_FIELD_LOCATOR = By.xpath("//input[@id='username']");
+    private static final By PASSWORD_FIELD_LOCATOR = By.xpath("//input[@id='password']");
+    private static final By LOGIN_BUTTON_LOCATOR = By.xpath("//input[@class='button r4 wide primary']");
+
+    public LoginPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void openURL() {
+        System.out.println("Open SaleseForce login page");
+        driver.get(URL);
+    }
+
+    public void inputUsername(String userName) {
```

Comment:
create class smth like Helpers
add method 
```
public static void clearAntInputValue(WebDriver driver, Locator locator) {
        driver.findElement(locator).clear();
        driver.findElement(locator).sendKeys(userName);
}
```
use it in all cases were u need clear and input e.g.
```
    public void inputPassword(String password) {
        System.out.println("Clear and populate Password field");
Helpers.cleatAndInputValue(driver, PASSWORD_FIELD_LOCATOR)
    }
```

Reply:
Done